File: packages/graph/__fixtures__/multi-lang-project/src/services/auth_service.py
from ..utils.hash import hash_password, verify_hash
from ..types import User, AuthToken

class AuthService:
    def __init__(self, secret: str):
        self._secret = secret

    def authenticate(self, username: str, password: str) -> AuthToken:
        hashed = hash_password(password)
        return {"token": hashed, "user": username}

    def validate_token(self, token: str) -> bool:
        return len(token) > 0

def create_auth_service(secret: str) -> AuthService:
    return AuthService(secret)

MAX_SESSIONS = 100

File: packages/graph/__fixtures__/multi-lang-project/src/services/auth-service.ts
import type { User, AuthToken } from '../types';
import { hashPassword } from '../utils/hash';

export class AuthService {
  private secret: string;

  constructor(secret: string) {
    this.secret = secret;
  }

  authenticate(username: string, password: string): AuthToken {
    const hashed = hashPassword(password);
    return { token: hashed, user: username };
  }
}

export const MAX_SESSIONS = 100;
